cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt

```diff
@@ -58,6 +59,7 @@ internal class SpConsentLibImpl(
 
     private val remainingCampaigns: Queue<CampaignModel> = LinkedList()
     private var currentNativeMessageCampaign: CampaignModel? = null
+    private var webview: IConsentWebView? = null
```

Comment:
@Nevazhnovu have a look at the `ViewManager.kt` file. The `ViewsManagerImpl` class has a reference to the SDK's main view 
```kotlin
    val mainView: ViewGroup?
        get() = weakReference.get()?.findViewById(R.id.content)
``` 

I recommend moving the logic there and using that reference, instead a new one in `SpConsentLib`.

Reply:
@andresilveirah It would totally make sense if we could embed `onBackPressed()` and make it internal. But currently, we still need to make that public in `SpConsentLibImpl.kt` in order for the user can access that.